src/graphics/program-lib/chunks/base.frag.js

```diff
@@ -3,6 +3,8 @@ uniform vec3 view_position;
 
 uniform vec3 light_globalAmbient;
 
+uniform float globalTextureBias;
```

Comment:
Wondering whether this should just be `textureBias`.

Reply:
Since texture bias could be specified per texture2D call I thought it was useful to highlight the fact this bias is used globally on all standard material textures.

Reply:
The shader is agnostic as to whether the `textureBias` value is updated once per scene or once per draw call though, right? To the shader, it's just a 'texture bias'.

Reply:
Right, but from the application perspective setting 'globalTextureBias' sets it for all the textures, globally. While naming it 'textureBias' wouldn't be wrong, I think there's value in making the fact this is a global uniform explicit. If I was reading code with a 'textureBias' variable, I'd assume there were one of these per texture.

Reply:
@mvaligursky do you have an opinion either way?

Reply:
and @willeastcott if you feel strongly ab it, I'm ok to go with 'textureBias'.

Reply:
I shall let @mvaligursky tie break.

Reply:
I agree with @willeastcott here. The shader just uses the uniform, that the engine sets up once per frame or once per draw call - shader knows no difference.

Reply:
noooooooooooo

Reply:
done